styleguide/derek/_partials/_mainnav.ejs

```diff
@@ -109,7 +109,7 @@
               <li class="navbar-topItem">
                 <span class="navbar-topLink dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Solutions</span>
                 <ul class="dropdown-menu navbar-dropDownMenu">
-                  <li class="navbar-dropDownItem"><a href="javascript:void(0)" column="1" columntitle="1" class="navbar-dropDownLink active">Managed Offerings</a></li>
+                  <li class="navbar-dropDownItem"><a href="javascript:void(0)" column="1" columntitle="1" class="navbar-dropDownLabel">Managed Offerings</a></li>
```

Comment:
@amberRrucker Can you switch the `<a>` to a `<span>` here, so it doesn't act like a link in the browser.